checks.d/sqlserver.py

```diff
@@ -190,6 +192,22 @@ def _conn_string(self, instance):
             conn_str += 'Integrated Security=SSPI;'
         return conn_str
 
+    def _conn_key_to_string(self, conn_key):
```

Comment:
can't we just add that to `_conn_string`? We could change it to `_conn_string(self, **kwargs)` and get the DB info with either `self._get_access_info(instance)` or `conn_key` depending on what's in `**kwargs`.

If we can avoid duplicating code that would be great!